.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
```

Comment:
:+1:

everything in `{lib,test}/browser` should have its env set to browser, maybe a few reporters. I think the references to browser stuff elsewhere is feature detection. was being lazy but I'll just fix those